index.d.ts

```diff
@@ -2222,16 +2230,26 @@ declare namespace Shopify {
     value: string;
     once_per_customer: boolean;
     usage_limit: number | null;
+    allocation_limit: number | null;
     customer_selection: PriceRuleCustomerSelection;
     prerequisite_saved_search_ids: number[];
     prerequisite_subtotal_range: IPriceRulePrerequisiteSubtotalRange | null;
-    prerequisite_shipping_price_range: string;
+    prerequisite_shipping_price_range: string | null;
```

Comment:
According to the docs this is not a string but an object like

```
{ less_than_or_equal_to: '10.0' }
```

when not `null`.

Reply:
Good spot! I didn't notice this one.

I've added an interface for `IPriceRulePrerequisiteShippingPriceRange`
```
interface IPriceRulePrerequisiteShippingPriceRange {
  less_than_or_equal_to: string;
}
```
and assigned it to the return type of `prerequisite_shipping_price_range`
```
interface IPriceRule {
  ...
  prerequisite_shipping_price_range: IPriceRulePrerequisiteShippingPriceRange | null;
  ...
}
```